packages/server-wallet/src/wallet/__test__/fixtures/test-channel.ts

```diff
@@ -39,6 +44,8 @@ export class TestChannel {
   public signingWalletB: SigningWallet = bobWallet();
   public startBals: [number, number];
   public channelNonce: number;
+  public finalFrom?: number;
```

Comment:
Can you put a docstring above `TestChannelArgs` to help explain `finalFrom`? Something like 

```ts
/**
 * Arguments for constructing a TestChannel
 *
 * @param finalFrom - a turnnumber where the channel bceoms finalized. All subsequent states willl be final and have this turn number
 */
interface TestChannelArgs {
  aBal?: number;
  bBal?: number;
  channelNonce?: number;
  finalFrom?: number;
  fundingStrategy?: FundingStrategy;
}

```